### DISCUSSION TITLE
Reconnect on lost connection

### DISCUSSION THREAD

**Discussion by trendzetter** [epoptes/epoptes#31] [Q&A]
I had to reboot the server after the clients had connected to it. The clients were not attempting to reconnect after the reboot. I had to reboot all the clients to reconnect them to the epoptes server.
I think it might be useful feature to retry the connection after a certain (configurable?) interval.
environment: epoptes-client on ltsp-pnp booted ubuntu 16.04.2
the server is debian8 epoptes 0.5.7

**Comment by trendzetter** [comment]
It should be implemented already. Looking into the cause on irc now.

**Comment by alkisg** [comment]
(the problem was an old epoptes-server version, 0.5.8)
